File: app/src/main/java/com/mappls/sdk/demo/java/activity/FeedbackUiActivity.java
package com.mappls.sdk.demo.java.activity;

import android.content.Intent;
import android.os.Bundle;
import android.view.Menu;
import android.view.MenuInflater;
import android.view.MenuItem;
import android.view.View;
import android.widget.Button;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;

import com.mappls.sdk.demo.R;
import com.mappls.sdk.demo.java.settingscreen.FeedbackSettingActivity;

public class FeedbackUiActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_feedback_ui);
        Button button = findViewById(R.id.feedback);
        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(FeedbackUiActivity.this, FeedbackWidgetActivity.class));
            }
        });
    }
    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        MenuInflater inflater = getMenuInflater();
        inflater.inflate(R.menu.widget_setting_menu, menu);

        return super.onCreateOptionsMenu(menu);

    }

    @Override
    public boolean onOptionsItemSelected(@NonNull MenuItem item) {
        if (item.getItemId() == R.id.widget_setting) {
            startActivity(new Intent(this, FeedbackSettingActivity.class));
        }
        return super.onOptionsItemSelected(item);

    }
}

File: app/src/main/java/com/mappls/sdk/demo/activity/widgets/FeedbackUiActivity.kt
package com.mappls.sdk.demo.activity.widgets

import android.content.Intent
import android.os.Bundle
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat
import com.mappls.sdk.demo.activity.widgets.feedbackUi.FeedbackWidgetActivity
import com.mappls.sdk.demo.databinding.ActivityFeedbackUiBinding

class FeedbackUiActivity : AppCompatActivity() {

    private lateinit var mBinding: ActivityFeedbackUiBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        mBinding = ActivityFeedbackUiBinding.inflate(layoutInflater)
        setContentView(mBinding.root)
        ViewCompat.setOnApplyWindowInsetsListener(mBinding.main) { v, insets ->
            val systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars())
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom)
            insets
        }
        mBinding.feedbackUiHeader.headerTitle.text = "Feedback Widget"
        mBinding.feedbackUiHeader.headerBack.setOnClickListener {
            finish()
        }

        mBinding.tvFeedbackWidget.setOnClickListener {
            startActivity(Intent(this, FeedbackWidgetActivity::class.java))
        }
    }
}